### ISSUE TITLE
How to integrate TOC in file 

### ISSUE THREAD

**Issue by parthi2929** [jupyterlab/jupyterlab-toc#32]
Hi

This is a great life savior for long files. While I could use on left pane, I also wish I could integrate this TOC on top of my markdown file. Is it possible? If not can you kindly integrate it?

**Comment by off99555** [comment]
Yes, this feature would be valuable. I want a script that can generate a TOC section on a cell for me. It doesn't need to be automatic. It can be manual every time I click a button to generate TOC for example.

**Comment by joelostblom** [comment]
To add a ToC cell in the notebook, it is possible to use the toc2 extension via launching the classic notebook from JupyterLab. To add a ToC cell to the exported HTML, it is possible to use `nbconvert` with the `html_toc` extension format provided by the toc2 extension, so you don't need to open the classic notebook. I wrote more detailed about this in [a SO answer](https://stackoverflow.com/a/59286150/2166823).
